### ISSUE TITLE
Add the ability to export garden connection configuration

### ISSUE THREAD

**Issue by scott-taubman** [beer-garden/beer-garden#1141]
Some users have expressed interested in the ability to export garden connection information as to ease the setup of child gardens.  This could be especially helpful in the STOMP connection case, where there can sometime be a long a list of custom headers defined that are tedious to re-enter every time.

This should be fundamentally similar to the job import and export effort and the API should be designed consistently with those endpoints.

Also of note, if this amounts to exporting / importing the entire garden definition, we should probably not include the system information in the export.

**Comment by jlrpnbbngtn** [comment]
After further discussion, it's agreed that:
1. Only the connection configuration for a specific garden will be exported/imported.
2. There needs to be strict validation of connection parameters, which will fall under another ticket (that blocks this one). There also will be a ticket to enable the import/export in the UI.

**Comment by jlrpnbbngtn** [comment]
Blocked by #1191
